raiden/api/rest.py

```diff
@@ -2251,3 +2272,12 @@ def create_light_client_payment(
             return ApiErrorBuilder.build_and_log_error(errors=str(e), status_code=HTTPStatus.NOT_FOUND, log=log)
         except UnhandledLightClient as e:
             return ApiErrorBuilder.build_and_log_error(errors=str(e), status_code=HTTPStatus.FORBIDDEN, log=log)
+
+    def post_unlock_payment_light(self, signed_tx: typing.SignedTransaction, token_address: typing.TokenAddress):
+        try:
+            self.raiden_api.unlock_payment_light(signed_tx, token_address)
+            return api_response(result=dict(), status_code=HTTPStatus.OK)
+        except RawTransactionFailed as e:
```

Comment:
You should keep consistency on the error handling, above you put something like: 
```python
try:
            self.raiden_api.register_secret_light(signed_tx)
            return api_response(result=dict(), status_code=HTTPStatus.OK)
        except InsufficientFunds as e:
            return api_error(errors=str(e), status_code=HTTPStatus.PAYMENT_REQUIRED)
        except (RawTransactionFailed, InvalidPaymentIdentifier) as e:
            return ApiErrorBuilder.build_and_log_error(errors=str(e), status_code=HTTPStatus.BAD_REQUEST, log=log)
        except Exception as e:
            return ApiErrorBuilder.build_and_log_error(errors=str(e), status_code=HTTPStatus.INTERNAL_SERVER_ERROR, log=log)
```
You should do the same here